modules/core/test/test_io.cpp

```diff
@@ -764,6 +768,10 @@ static void test_filestorage_basic(int write_flags, const char* suffix_name, boo
         if (useMemory) {
             EXPECT_EQ(0, cv::norm(_rd_in, _rd_out, NORM_INF));
         }
+        if (testReadWrite && !useMemory && !generateTestData) {
+            EXPECT_EQ(0, remove(name.c_str()));
```

Comment:
Origin of this code is 4.x branch.

To avoid merge conflicts in the future prefer to keep the code formatting from 4.x:

```
if (testReadWrite && !useMemory && !generateTestData)
{
    EXPECT_EQ(0, remove(name.c_str()));
}
```

---

If you want to change code formatting then you should go to 4.x first.